prdoc/pr_1818.prdoc

```diff
@@ -0,0 +1,14 @@
+title: FRAME pallets warning for unchecked weight witness
+
+doc:
+  - audience: Core Dev
+    description: |
+      FRAME pallets now emit a warning when a call uses a function argument that starts with an underscore in its weight declaration.
+
+migrations:
+  db: [ ]
+  runtime: [ ]
+
+host_functions: []
+
+crates: [ ]
```

Comment:
do we need to list changed crates here?

Reply:
I tried to list something here, but it wants me to put some semver things, and i dont think we can do that yet, or @chevdor ?

Reply:
Okay I talked to Will and now filled in one crate.